sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Generated/SdkInfo_DataFactoryManagementClient.cs

```diff
@@ -1,40 +0,0 @@
-
```

Comment:
please do not remove this file

Reply:
Ok.
Just note that it was removed by autorest (both for me and for Wenbo).